dali/pipeline/operators/op_schema.h

```diff
@@ -173,6 +174,14 @@ class DLL_PUBLIC OpSchema {
     return *this;
   }
 
+  /**
+   * @brief Notes that this operator is internal to DALI backend (and shouldn't be exposed in Python API)
+   */
+  DLL_PUBLIC inline OpSchema& InternalOp() {
```

Comment:
I'd rename it to `SetInternal`, `MakeInternal` or sth along these lines - InternalOp sounds like a getter to some internal op.

Reply:
Done